src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -672,6 +666,7 @@ namespace winrt::TerminalApp::implementation
         // Add the new tab to the list of our tabs.
         auto newTabImpl = winrt::make_self<Tab>(profileGuid, term);
         _tabs.Append(*newTabImpl);
+        _mruTabActions.Append(newTabImpl->SwitchToTabCommand());
```

Comment:
is this line necessary? Won't the new tab be appended automatically in `_UpdatedSelectedTab`?

I suppose it doesn't really matter. The redundancy is nice, especially if we ever have a way to open new tabs without automatically switching to them

Reply:
I kinda thought of `_UpdatedSelectedTab` as the function that does things when a Tab is focused, so it assumes that the Tab already exists in the list when it's called. That's why for MRU, `_UpdatedSelectedTab` doesn't append a new tab, it drags and drops the focused tab to the top of MRU. Then the adding of a new Tab/SwitchToTabCommand should be done by `CreateNewTabFromSettings`.

Reply:
Okay that's fine.